src/lib/autoinstall/presenters/partition.rb

```diff
@@ -86,6 +86,19 @@ def available_lvm_groups
         names.map { |n| n.delete_prefix("/dev/") }
       end
 
+      # Updates the Autoinst partition section
+      #
+      # Redefines the Section#update method for restricting the section cleanup to given attributes.
+      #
+      # @param values [Hash] Values to update
+      def update(values)
```

Comment:
Why redefining this in `Partition`? Cannot we just change `Section#update` to work like this always?

Reply:
A kind of leftover, because before reaching this approach I was using/playing with a more "partition oriented" solution (don't ask me which :wink:). But sure, this can be moved to the `Section#update`.